dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java

```diff
@@ -21,18 +21,9 @@
 import static org.apache.dolphinscheduler.common.Constants.RESOURCE_VIEW_SUFFIXS_DEFAULT_VALUE;
 import static org.apache.dolphinscheduler.common.Constants.YYYYMMDDHHMMSS;
 
-import java.io.BufferedReader;
-import java.io.BufferedWriter;
-import java.io.ByteArrayOutputStream;
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.io.StringReader;
-import java.io.UnsupportedEncodingException;
+import java.io.*;
```

Comment:
Please configure checkstyle and code-style, ctrl+L will automatic to format the code.

Reply:
Is this mandatory?

I personally don't like controls like AutoFormat

After a lot of time, there will be uncontrollable formatting

Reply:
It is mandatory, and there is the reason why we need checkstyle.[1]

Actually, checkstyle automatic format will make the code more controllable.
Maybe when we review code, it will add a little work, but the format and clear code will help user to read it more easier and quickly.

For example in this pr,
There is a lot unnecessary changes about import in this pr, if we use checkstyle auto import, there will not have a lot of changes about import.

[1] https://lists.apache.org/thread.html/ra65d23640728b12a8a1102495202ff066accfe99e80e5c694e093d49%40%3Cdev.dolphinscheduler.apache.org%3E